wagtail_airtable/management/commands/import_airtable.py

```diff
@@ -0,0 +1,239 @@
+        Given an 'app_name.model_name' string, return the model class
+        """
+        app_label, model_name = model_path.split('.')
+        return ContentType.objects.get_by_natural_key(app_label, model_name).model_class()
+
+    def get_model_serializer(self, serializer_string):
+        location, serializer_name = serializer_string.rsplit(".", 1)
+        module = import_module(location)
+        serializer_class = getattr(module, serializer_name)
+        return serializer_class
+
+    def handle(self, *args, **options):
+        """
+        Runs the management command with the app_name.ModelName as parameters.
+        ie. python manage.py import_airtable pages.HomePage creditcards.CreditCard
+
+        This will handle Wagtail Pages (with page revisions) along with standard
+        Django models.
+
+        Models passed in to the command go through a quick validation to ensure they
+        exist.
+
+        Every model is then looped through, and a `settings.AIRTABLE_IMPORT_SETTINGS`
+        is searched for based on the models label (ie. pages.HomePage). These
+        settings are used to connect to a certain Airtable Base (a set of spreadsheets),
+        the name of the table to use, a unique identifier (used for connecting previously
+        unrelated Airtable records to Django objects), and a serializer for validating
+        incoming data from Airtable to make it work with the Django field types.
+
+        Within each model loop also contains an `airtable.get_all()` command which
+        will get all the data from the Airtable spreadsheet and load it into a
+        list of dictionaries.
+
+        Then every record is iterated over and 3 actions are taken:
+            1. Look for an existing model object by its airtable_record_id.
+               Update if found. Skip to step 2 if not found.
+            2. Search for a model object by its unique identifier.
+               If found, update the object. Skip to step 3 if not found.
+            3. Create a new object. This step is very susceptible to fail based on the
+               model type and fields in the serializer. Wagtail pages cannot be created
+               from Airtable records as there's missing data such as depth and parent
+               pages. And so Wagtail pages are skipped entirely in step 3.
+        """
+        models = []
+        for label in options['labels']:
+            label = label.lower()
+            if '.' in label:
+                # interpret as a model
+                try:
+                    model = self.get_model_for_path(label)
+                except ObjectDoesNotExist:
+                    raise CommandError("%r is not recognised as a model name." % label)
+
+                models.append(model)
+
+        created = 0
+        updated = 0
+        skipped = 0
+        for model in models:
+            # Wagtail models require a .save_revision() call when being saved.
+            is_wagtail_model = hasattr(model, 'depth')
```

Comment:
Yep, good find. I refactored this into a method that's now using `if issubclass(model, Page):` because `model` is the Model and not an instance of one. This will help determine if a model is a Page or not. Tests still pass with this. 👍